Jin

uzzzzin opened the pull request:
1주차 pull request ... test

kimjw395 committed:
-- 연산자(전위, 후위) 구현
end() 구현
begin() 수정
복사, 이동 연산자 구현
복사, 이동 생성자 구현
main 함수에 swap(이동 연산자 사용하여) 구현
CopyShallow함수와 CopyDeep 함수 구현
  - 이동, 복사 생성자 구현
  - 대입, 이동 대입 연산자 구현
  - CopyShallow함수와 CopyDeep 함수 구현

Commit:
8월 22일 동적배열 클래스 - 복사, 이동 생성자 및 대입 연산자 구현

jeontigger commented on Jin/exrl/CArray.cpp:
```diff
@@ -0,0 +1,108 @@
+using std::cout;
+using std::endl;
+
+// ������ �Ҹ���
+// ũ�� �缳�� �Լ�
+// ����, �̵� ������
+// ����, �̵� ���� ������
+// deep, shallow copy �Լ�
```
이건 헤더를 작성하기 위한 코드기 때문에 헤더에서 하는 것이 더 좋아보임
헤더를 바탕으로 CPP를 구현하는데, 그 때 함수 내용을 다시 슈도코드 작성할 때 CPP에 작성한다고 보면 됨

Commit:
#1 템플릿 링크드리스트 구현

jeontigger committed:
Merge branch 'main' into Jin

Commit:
#1 메모리 누수를 해결
crtdbg 헤더를 통해서
_CrtSetDbgFlag(_CRTDBG_ALLOC_MEM_DF | _CRTDBG_LEAK_CHECK_DF);
_CrtSetBreakAlloc(154);

Commit:
Merge branch 'Jin' of https://github.com/AssortRockStudy/followUp into Jin
